packages/labs/cli/src/test/gen/react_test.ts

```diff
@@ -24,20 +32,38 @@ test.before(() => {
   });
 });
 
-test('stub test', async () => {
+test('basic wrapper generation', async () => {
+  const inputPackage = 'test-project';
+  const outputPackage = path.join(outputFolder, inputPackage + '-react');
+
   const cli = new LitCli(
-    ['labs', 'gen', '--framework', 'react', '--package', 'test-project'],
+    [
+      'labs',
+      'gen',
+      '--framework',
+      'react',
+      '--package',
+      inputPackage,
+      '--out',
+      outputFolder,
+    ],
     {
       console: cliConsole,
     }
   );
   await cli.run();
 
-  const output = outputStream.text;
-
   assert.equal(errorStream.buffer.length, 0);
 
-  // TODO: change to actual test once more is fleshed out; for now the stub
-  // command just outputs analyzed modules
-  assert.match(output, 'element-a.ts');
+  const wrapperFile = fs.readFileSync(
+    path.join(outputPackage, '/src/element-a.ts')
+  );
+  assert.ok(wrapperFile.length > 0);
+
+  await installPackage(outputPackage, {
+    'test-project': inputPackage,
+    '@lit-labs/react': '../react',
+  });
+
+  await buildPackage(outputPackage);
```

Comment:
There are no asserts here because the idea is they just shouldn't error? If so, add a comment.

Reply:
Added an assert and TODO to add golden tests in this spot.